Bottom: a32e4d9e22ad1568437c981193748b529619fbbd
Top:    6f7546b7ae5dbeb63bf6691d649100282a65e352
Author: Andrew Cooper <andrew.cooper@novatechweb.com>
Date:   2018-08-29 09:24:23 -0500

Refresh of add-buildbot-role

---
diff --git a/roles/buildbot-master/tasks/main.yml b/roles/buildbot-master/tasks/main.yml
index f2d530d1..b68c44c2 100644
--- a/roles/buildbot-master/tasks/main.yml
+++ b/roles/buildbot-master/tasks/main.yml
@@ -17,19 +17,18 @@
     group: '{{ buildbot_uid }}'
     owner: '{{ buildbot_uid }}'
   loop:
-    - buildbot.tac
     - master.cfg
     - openembedded.py
     - ptxdist.py
 
 - name: Populate config templates
-  copy:
-    src: "{{ item }}"
+  template:
+    src: "{{ item }}.j2"
     dest: "{{ config_hostdir }}/{{ item }}"
     group: '{{ buildbot_uid }}'
     owner: '{{ buildbot_uid }}'
   loop:
-    - buildbot.tac.j2
+    - buildbot.tac
 
 - name: Create secrets volume
   file:
